docs/pbc/all/tax-management/domain-model-and-relationships/tax-entity-digrams.md

```diff
@@ -0,0 +1,62 @@
+---
```

Comment:
See comment in the `pbc_all_sidebar.yml`. This file must be put as `domain-model-and-relationships.html` under the `tax-management` folder